pyaerocom/colocation.py

```diff
@@ -169,9 +172,15 @@ class can handle different methods with different inputs)
         gridded_data.base_year = update_baseyear_gridded
 
     if regrid_res_deg is not None:
-        gridded_data_ref = gridded_data_ref.regrid(lat_res_deg=regrid_res_deg,
-                                                   lon_res_deg=regrid_res_deg,
-                                                   scheme=regrid_scheme)
+        if not isinstance(regrid_res_deg, dict):
+            if not isnumeric(regrid_res_deg):
+                raise ValueError('Invalid input for regrid_res_deg. Need integer '
+                                 'or dict specifying lat and lon res')
+            regrid_res_deg = dict(lat_res_deg=regrid_res_deg,
+                                  lon_res_deg=regrid_res_deg)
+
+        gridded_data_ref = gridded_data_ref.regrid(scheme=regrid_scheme,
+                                                   **regrid_res_deg)
```

Comment:
This logic is repeated in `colocate_gridded_ungridded` (except from operating on gridded_data instead of gridded_data_ref). Move it into a function?

Reply:
Sounds good, will do that!